style_editor.html

```diff
@@ -5618,7 +5618,7 @@
 class MultiTransitionEffectLClass extends STYLE {
   constructor(TRANSITION, EFFECT_ARG, N) {
     super("Trigger transitions on an effect.", arguments);
-    this.add_arg("TRANSITION", "TRANSITION", "from EFFECT_COLOR T");
+    this.add_arg("TRANSITION", "TRANSITION", "from Effect Color back to Base");
```

Comment:
This is a bit confusing, because the transition really goes from base color to base color.
(There is no effect color here)

Reply:
so... just "Transition Effect"?
This would apply to regular TransitionEffectL as well.

Also while we're here, MultiTransitionEffectL's "Simultaneous Effects" (last argument) is offset by 1.
If it's set to 4, 3 effects are allowed before resetting to accomodate a new one.
I edited the math in the class to fix it.